src/features/Application/ManageApplicationContainer.tsx

```diff
@@ -364,66 +348,221 @@ class ManageApplicationContainer extends React.Component<
           component={FormikElements.Error}
-          name="application.shortAnswer.question1"
+          name="hacker.application.shortAnswer.question1"
         />
         <FastField
-          name={'application.shortAnswer.question2'}
+          name={'hacker.application.shortAnswer.question2'}
           component={FormikElements.LongTextInput}
           label={CONSTANTS.QUESTION2_REQUEST_LABEL}
-          value={fp.values.application.shortAnswer.question2}
+          value={fp.values.hacker.application.shortAnswer.question2}
           maxLength={2000}
           required={true}
         />
         <ErrorMessage
           component={FormikElements.Error}
-          name="application.shortAnswer.question2"
+          name="hacker.application.shortAnswer.question2"
         />
         <FastField
-          name={'application.shortAnswer.comments'}
+          name={'hacker.application.shortAnswer.comments'}
           component={FormikElements.LongTextInput}
           label={CONSTANTS.COMMENTS_REQUEST_LABEL}
-          value={fp.values.application.shortAnswer.comments}
+          value={fp.values.hacker.application.shortAnswer.comments}
           maxLength={500}
           required={false}
         />
         <ErrorMessage
           component={FormikElements.Error}
-          name="application.shortAnswer.comments"
+          name="hacker.application.shortAnswer.comments"
+        />
+        <Flex
+          flexDirection={'row'}
+          alignItems={'center'}
+          justifyContent={'space-between'}
+        >
+          <ResetBtn isLoading={false} disabled={this.state.submitting}>
+            Back
+          </ResetBtn>
+          <SubmitBtn
+            isLoading={this.state.submitting}
+            disabled={this.state.submitting}
+          >
+            Next
+          </SubmitBtn>
+        </Flex>
+      </Form>
+    );
+  }
+
+  /**
+   * Renders the accommodation section of the application.
+   * @param fp the formik props.
+   */
+  private renderAccommodationFormik(fp: FormikProps<any>) {
+    return (
+      <Form
+        onKeyDown={this.onKeyDown}
+        onSubmit={fp.handleSubmit}
+        onReset={fp.handleReset}
+      >
+        <FastField
+          name={'hacker.application.accommodation.shirtSize'}
+          label={CONSTANTS.SHIRT_SIZE_LABEL}
+          component={FormikElements.Select}
+          options={getOptionsFromEnum(ShirtSize)}
+          required={true}
+          value={fp.values.hacker.application.accommodation.shirtSize}
+        />
+        <ErrorMessage
+          name={'hacker.application.accommodation.shirtSize'}
+          component={FormikElements.Error}
+        />
+        {/* This fixes the issue with going back somehow, so leave it here temporarily */}
+        <div />
+        <FastField
+          name={'hacker.application.accommodation.impairments'}
+          component={FormikElements.LongTextInput}
+          label={'Do you have any impairments that we should know about?'}
```

Comment:
Same with this, most of the other labels have been abstracted to the constants file.